Application/src/view/visuals/Draw.java

```diff
@@ -1,5 +1,6 @@
 package view.visuals;
 
+import com.sun.org.apache.xerces.internal.impl.xpath.XPath;
```

Comment:
what does this do? it is confusing my IDE

Reply:
It got automatically added, not sure

Reply:
@IsabelleTornqvist remove it please, my ide only runs when i remove it

Reply:
+1